src/ocaml/typing/typecore.ml

```diff
@@ -2146,16 +2142,15 @@ let check_let_mutable (mf : mutable_flag) env ?restriction vbs =
          - Mutables are not restricted here according to [restriction] *)
       Language_extension.assert_enabled ~loc Let_mutable ();
       match restriction, vb.pvb_pat.ppat_desc, vbs with
-      | _, _, _ :: _ -> raise (Error (loc, env, Unexpected_mutable In_group))
-      | Some r, _, _ -> raise (Error (loc, env, Unexpected_mutable r))
+      | _, _, _ :: _ -> raise_error (error (loc, env, Unexpected_mutable In_group))
+      | Some r, _, _ -> raise_error (error (loc, env, Unexpected_mutable r))
       | None, Ppat_var _, [] -> ()
       | None, (Ppat_constraint ({ppat_desc=Ppat_var _}, _, _)), [] -> ()
-      | None, _, [] -> raise (Error (loc, env, Illegal_mutable_pat))
+      | None, _, [] -> raise_error (error (loc, env, Illegal_mutable_pat))
```

Comment:
`check_let_mutable` should get reviewed

Reply:
I'm not sure - seems like the conflicts resolution is trivial since one side is empty?

Reply:
You mean, if I write `raise (Error (loc, env, Illegal_mutable_pat))` there, this will still build, but then won't gracefully recover from error?